docs/apidocs/qiskit_nature.settings.rst

```diff
@@ -1,7 +1,7 @@
-﻿Settings
```

Comment:
I am not sure why some of these files, like this and the utils one, are changed at all since that code is not moving right. The stubs here, whether auto-generated by sphinx as starting point or not, need cleaning up at some point. These stubs were talked about in an earlier comment so I'll leave it at that.